src/widgets/Schedule.ts

```diff
@@ -85,7 +85,11 @@ export default class Schedule extends Widget<ScheduleConfig> {
     const url = this.getUrl();
     transport.get(url, {},
       (data: IPlainObject[]) => {
-        self.events = data.map((event: IPlainObject) => {
+        var events = data;
```

Comment:
Please, use `let` instead of `var`. `var` is too generic and too javascript-ish :)

Reply:
Oh interesting. Thanks for your feedback. Will change that.